tools/gfx-unit-test/shader-cache-tests.cpp

```diff
@@ -1255,92 +1252,93 @@ namespace gfx_test
         test.run();
     }
 
-    SLANG_UNIT_TEST(singleEntryShaderCacheD3D12)
+    SLANG_UNIT_TEST(shaderCacheSingleEntryD3D12)
     {
         runTestImpl(shaderCacheTestImpl<SingleEntryShaderCache>, unitTestContext, Slang::RenderApiFlag::D3D12);
     }
 
-    SLANG_UNIT_TEST(singleEntryShaderCacheVulkan)
+    SLANG_UNIT_TEST(shaderCacheSingleEntryVulkan)
     {
         runTestImpl(shaderCacheTestImpl<SingleEntryShaderCache>, unitTestContext, Slang::RenderApiFlag::Vulkan);
     }
 
-    SLANG_UNIT_TEST(multipleEntryShaderCacheD3D12)
+    SLANG_UNIT_TEST(shaderCacheMultipleEntryD3D12)
     {
         runTestImpl(shaderCacheTestImpl<MultipleEntryShaderCache>, unitTestContext, Slang::RenderApiFlag::D3D12);
     }
 
-    SLANG_UNIT_TEST(multipleEntryShaderCacheVulkan)
+    SLANG_UNIT_TEST(shaderCacheMultipleEntryVulkan)
     {
         runTestImpl(shaderCacheTestImpl<MultipleEntryShaderCache>, unitTestContext, Slang::RenderApiFlag::Vulkan);
     }
 
-    SLANG_UNIT_TEST(multipleEntryPointShaderCacheD3D12)
+    SLANG_UNIT_TEST(shaderCacheMultipleEntryPointD3D12)
     {
         runTestImpl(shaderCacheTestImpl<MultipleEntryPointShader>, unitTestContext, Slang::RenderApiFlag::D3D12);
     }
 
-    SLANG_UNIT_TEST(multipleEntryPointShaderCacheVulkan)
+    SLANG_UNIT_TEST(shaderCacheMultipleEntryPointVulkan)
     {
         runTestImpl(shaderCacheTestImpl<MultipleEntryPointShader>, unitTestContext, Slang::RenderApiFlag::Vulkan);
     }
 
-    SLANG_UNIT_TEST(shaderFileImportsShaderCacheD3D12)
+    SLANG_UNIT_TEST(shaderCacheFileImportsD3D12)
     {
         runTestImpl(shaderCacheTestImpl<ShaderFileImportsShaderCache>, unitTestContext, Slang::RenderApiFlag::D3D12);
     }
 
-    SLANG_UNIT_TEST(shaderFileImportsShaderCacheVulkan)
+    SLANG_UNIT_TEST(shaderCacheFileImportsVulkan)
     {
         runTestImpl(shaderCacheTestImpl<ShaderFileImportsShaderCache>, unitTestContext, Slang::RenderApiFlag::Vulkan);
     }
 
-    SLANG_UNIT_TEST(specializationArgsShaderCacheD3D12)
+    SLANG_UNIT_TEST(shaderCacheSpecializationArgsD3D12)
     {
         runTestImpl(shaderCacheTestImpl<SpecializationArgsEntries>, unitTestContext, Slang::RenderApiFlag::D3D12);
     }
 
-    SLANG_UNIT_TEST(specializationArgsShaderCacheVulkan)
+    SLANG_UNIT_TEST(shaderCacheSpecializationArgsVulkan)
     {
         runTestImpl(shaderCacheTestImpl<SpecializationArgsEntries>, unitTestContext, Slang::RenderApiFlag::Vulkan);
     }
-
-    SLANG_UNIT_TEST(cacheEvictionPolicyD3D12)
+#if 0
+    SLANG_UNIT_TEST(shaderCacheEvictionPolicyD3D12)
```

Comment:
These tests may be out of date but we need to make sure the eviction policy is tested.